Branch-SDK/Branch-SDK/Requests/BranchSetIdentityRequest.m

```diff
@@ -57,7 +57,9 @@ - (void)processResponse:(BNCServerResponse *)response error:(NSError *)error {
     preferenceHelper.identityID = response.data[BRANCH_RESPONSE_KEY_BRANCH_IDENTITY];
     preferenceHelper.userUrl = response.data[BRANCH_RESPONSE_KEY_USER_URL];
     preferenceHelper.userIdentity = self.userId;
-    preferenceHelper.sessionID = response.data[BRANCH_RESPONSE_KEY_SESSION_ID];
+    if (response.data[BRANCH_RESPONSE_KEY_SESSION_ID]) {
```

Comment:
Why would this be possible? This seems to indicate a problem elsewhere. Kinda similar to that type checking thing, it doesn't seem like the best idea to just wrap everything in if statements. I know it's just the one now, but you could really do the same for any of these.